index.js

```diff
@@ -158,9 +158,7 @@ module.exports = {
       },
     },
     all: {
-      plugins: [
-        'react',
-      ],
+      plugins: ['react'],
```

Comment:
please revert unrelated formatting changes; if perhaps you're running prettier on repos that do not use it, please don't do that either :-)